static/templates/left_sidebar.hbs

```diff
@@ -69,6 +55,39 @@
                 <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
         </ul>
+
+        <div id="private_messages" class="zoom-out hidden-for-spectators">
```

Comment:
`sidebar-private-messages-container` is a more reasonable name for this.

Reply:
OK, would go further with this name.